stories/Card.stories.mdx

```diff
@@ -0,0 +1,104 @@
+import { Meta, Canvas, Story, ArgsTable } from '@storybook/addon-docs';
+import { linkTo } from '@storybook/addon-links';
+import { Card, Header, Footer, Text } from '../src'; // components you used in the story
+
+
+<Meta title="Layout/Card" component={Card} />
+
+# Card
+
+The card will be used as a general interface to provide general structure to a UI component.
+It comes with very basic styling, but has a ton of customization. It includes options for headers and footers.
+The card component is made in such a way that it can be used as a container for other components.
+
+## Functionality
```

Comment:
Functionality should always come *after* the Setup. Also, the Setup should only be a basic example of how to get something working. The Functionality talks about all the other use cases.